include/boost/url/detail/config.hpp

```diff
@@ -110,4 +110,10 @@ using pos_t = size_t;
 #endif
 #endif
 
+#if defined(BOOST_URL_DOCS)
+#define BOOST_URL_PCT_STRING_VIEW string_view
```

Comment:
I think you have to do this in the Jamfile, not like this

Reply:
Good idea. Done.